### ISSUE TITLE
Move from ubuntu-20.04 to ubuntu-22.04 GitHub Actions runners

### ISSUE THREAD

**Issue by ShubhamTatvamasi** [kubearmor/KubeArmor#1433]
Currently, most of our GitHub actions CI jobs are running on `ubuntu-20.04` runners, now it's time we move to `ubuntu-22.04`.

**Comment by shreshthgoyal** [comment]
Hey @ShubhamTatvamasi I would like to work on this. Could you please assign this to me?

**Comment by AniketNS** [comment]
Hey @ShubhamTatvamasi, I would like to work on this issue. Can you please assign this to me?

**Comment by pushkarm029** [comment]
@Srijan-SS02 are currently working on this?

**Comment by Srijan-SS02** [comment]
> @Srijan-SS02 are currently working on this?

Yup I am, it is just a little busy with my semester exams but still in touch on slack.

**Comment by joelwilliam2005** [comment]
Hi @ShubhamTatvamasi,
I'm interested to work on this issue.
I noticed that there hasn't been an update on this issue for a while. Please let me know if anyone is working on this issue, else I'll be happy to take it over.

**Comment by AryanBakliwal** [comment]
Closing this as all our workflows now run on Ubuntu 22.04 or higher.
